Changes

```diff
@@ -1,5 +1,37 @@
 Release history of Zonemaster
 
+v2021.1 2021-05-31 (public release version)
+
+ [Features]
+ - Adds implementation of test cases DNSSEC15, DNSSEC16 and DNSSEC17 for
+   controll of CDS and CDNSKEY. See Zonemaster-Engine for this feature.
```

Comment:
"Controll" is misspelled. Though I believe "checking" would be better than "control of".

Reply:
Fixed. Pleaes review again.